TYPO3.Neos/Documentation/CreatingASite/NodeTypes/NodeTypeDefinition.rst

```diff
@@ -119,6 +119,32 @@ The following options are allowed:
     Alternatively the class of a node label generator implementing
     ``TYPO3\TYPO3CR\Domain\Model\NodeLabelGeneratorInterface`` can be specified as a nested option.
 
+``fusion``
```

Comment:
The below mentioned request to change the way the configuration is setup obviously makes a change here necessary.